template/v1/Dockerfile

```diff
@@ -57,7 +57,7 @@ RUN rsync -a ${DIRECTORY_TREE_STAGE_DIR}/ / && \
 
 # CodeEditor - download the extensions
 RUN mkdir -p /etc/code-editor/extensions && \
-    while IFS= read -r url; do \
+    while IFS= read -r url || [ -n "$url" ]; do \
```

Comment:
this change ensures that the last line is also processed (regardless of a new line at end)